haiku-apps/bemini/bemini-1.0.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="Native Gemini client for Haiku"
+DESCRIPTION="Bemini is a native Gemini browser for Haiku. \
+It's a python3 program that uses Haiku APIs through Haiku-PyAPI, \
+and two python libraries, trimgmi and gusmobile."
+HOMEPAGE="https://github.com/coolcoder613eb/Bemini"
+COPYRIGHT="2024 Elozor Bruce (coolcoder613eb)"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/coolcoder613eb/Bemini/archive/298e3a927d70ebbba1b1d18285ce25821d74da16.zip"
+CHECKSUM_SHA256="d594698c69216b38d6966936d19b3a44dc828f06a0702ba8ff20386bcf279bdf"
+SOURCE_DIR="Bemini-298e3a927d70ebbba1b1d18285ce25821d74da16"
+pythonVersion=3.10
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	bemini = $portVersion
+	app:Bemini = $portVersion
+	cmd:Bemini
+	"
+REQUIRES="
+	haiku
+	haiku_pyapi_python310
+	pip_python310
+	"
+BUILD_REQUIRES="
+    cmd:pip
+    cmd:git
+    "
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp main.py $appsDir/Bemini
+	chmod +x $appsDir/Bemini
+	pip install --target $prefix/lib/python$pythonVersion/vendor-packages/ trimgmi git+https://git.sr.ht/~rwa/gusmobile
```

Comment:
Please package a specific version in dev-python, and add as requirement of this package.